src/design-system/symbols/generated/index.ts

```diff
@@ -4616,1354 +4616,4 @@ export default {
       viewBox: { width: 32.83984375, height: 26.66015625 },
     },
   },
-  '1.circle': {
```

Comment:
wasnt this daniel;s pr?

Reply:
Yeah. I'm waiting for that to get merged and then rebase.